### ISSUE TITLE
Continue watching

### ISSUE THREAD

**Issue by pawel54321** [spocky/miproja1#98]
Hello,

I have question about continue watching. In original "continue watching" working (Google TV), but Your Launcher don't show videos (I see only slider active, but nothing happen).

**Comment by spocky** [comment]
Not sure I understand : do you mean there's not thumbnail in watch next on Chromecast ?
What's your Projectivy version ?

**Comment by pawel54321** [comment]
> Not sure I understand : do you mean there's not thumbnail in watch next on Chromecast ? What's your Projectivy version ?

Newest 4.54.

The problem is that movies are not displayed in continue watching. For example, on the original Chromecast we have YouTube continue watching (desktop), but not in projectivity Launcher. 
There is a slider in projectivity Launcher to activate, but if it is turned on, it does not show it anyway

**Comment by danwie** [comment]
I have the same issue. I'm using "Chromecast with Google TV". Continue watching is active but no content is visible.

**Comment by pawel54321** [comment]
Problem dotyczy też innych serwisów nie tylko YouTube na przyklad: Rakuten TV but not only

**Comment by pawel54321** [comment]
The problem also affects other websites, not only YouTube, for example: Rakuten TV, but not only

**Comment by danwie** [comment]
> I have the same issue. I'm using "Chromecast with Google TV". Continue watching is active but no content is visible.

I forgot to say that I'm currently only using Jellyfin and SmartTube. Maby they are not supported?

**Comment by danwie** [comment]
It might be problem with permissions. It says when you first run Projectivy Launcher that a permission is needed in order to use Continue watching. It then takes you to settings - but I couldn't find the needed setting to change, and then the Projectivy Launcher continues with other settings. Could it be that some setting needs to be set in order to use this feature? What setting?

**Comment by dunctait** [comment]
+1

**Comment by SJ1O** [comment]
Is stremio, smartube, netflix supported in continue watching / Watch next? Sorry if this has been asked a million times. I have googled it and see alot of people saying it works for them and some saying it doesn't. No problem if it doesn't was just curious. Great app thank you sir

**Comment by Feature-requester-general** [comment]
I have something related. Smarttube does not show up in the channel, nor does Crunchyroll. Other streaming services (Max, Prime,  etc.) do.

**Comment by rainman74** [comment]
I can't quite confirm the problem with 4.60.

It works with SmartTube, Spotify and Kodi (Maven's fork specifically designed for background play).

It doesn't work with e.g. Amazon Prime and Plex, but that doesn't work with the standard Android launcher either! So this is not a problem for Projectivy. The apps are closed automatically and do not run as a background task and therefore do not allow background play.

SmartTube
![Image](https://github.com/user-attachments/assets/0a3141da-fd69-4c8e-9d8b-fe3200996b21)

Spotify
![Image](https://github.com/user-attachments/assets/8d02f290-a067-480b-987c-4cc40b4afbdd)

**Comment by spocky** [comment]
For most Channel related stuff, Google Tv use their own (close source) behavior, not their previous standard API. Which means that things working on Android Tv might not work anymore on Google Tv (I believe a few apps don't fill their channels anymore on Google Tv, or maybe it's the OS that doesn't honor their request to fill programs as it doesn't use them anymore).
On recent Projectivy versions, everything should be in sync with what Android Tv does. 
Regarding Google Tv, this is rather mysterious (and unfortunately no other API has surfaced, as it's for their own use)

**Comment by pawel54321** [comment]
Hello spocky!
Launcher has been updating automatically recently. It's great that it's still supported, it looks great now. However, I've noticed that the icons for Max and Prime Video services have disappeared in the latest version in "watch next". I also have a question about the title and description, whether it's possible to customize it, because now I'm not sure. The description is presented differently, sometimes just the description of the film and sometimes the year and sometimes the length of the film.

**Comment by spocky** [comment]
I don't believe there are regression in Projectivy concerning icons in the  watch next Are you sure 4.54 still shows icons while 4.6x doesn't ?
Year/length are fixed in the latest 4.67.

**Comment by spocky** [comment]
Sorry it took so long to figure out. Will be fixed on 4.70

**Comment by tboneduggins** [comment]
Omfg I've unsubscribed by the link at the bottom of this email like 5
times!!

How do I stop these notification!?

Please help thanks

Travis Boone
